### ISSUE TITLE
Authorization

### ISSUE THREAD

**Issue by noyainrain** [noyainrain/meetling#4]
For editing the global settings we need a means of authorization.

Depends on #2 .

**Comment by noyainrain** [comment]
Draft:

``` python
class Settings:
    """
    .. describe:: staff

       Staff users.
    """

    def edit(self, **attrs):
        """
        .. http:post:: /api/settings

           ``{attrs...}``

           Edit the attributes given by *attrs* and return the updated settings.

           Permission: staff user.
        """

class PermissionError(Exception):
    """Returned if the current user is not allowed to perform an action."""
```

**Comment by noyainrain** [comment]
Implemented in ff1511e795d432a0504f105874bff5425215a5e6 .
